tests/acceptance-tests/src/acceptance/rds_broker_test.go

```diff
@@ -0,0 +1,22 @@
+package acceptance_test
+
+import (
+	"github.com/cloudfoundry-incubator/cf-test-helpers/cf"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gbytes"
+	. "github.com/onsi/gomega/gexec"
+)
+
+var (
+	brokerName = "rds"
```

Comment:
Should we call this "postgres" so that users are able to easily identify what it is?